### PR TITLE
Release notes for GraFx Publisher 2024-04-23

### PR DESCRIPTION
**Pull request by seancrowe**

### CODE REVIEW

File: docs/release-notes/releaseposts/2024042301.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<!-- more -->
+
+## Improvements
+
+- Strengthened security of REST API
+
+## Fixes
+
+- Fixed issue when drag and drop failed from external assets panel
+- Fixed error related to naming issue during certain API call
```
Comment (jeroencosyn) [inline docs/release-notes/releaseposts/2024042301.md:19]: Should be "Fixed error related to naming issue during certain API call**s**" according to Lina's post.

**Comment by jeroencosyn** [comment]
It doesn't show any intro text above "Continue reading". The other release notes do show something. But don't know if it's really needed.
image

**Review by jeroencosyn — DISMISSED** [review]

**Review by BramVerniest — APPROVED** [review]
LGTM

**Review by BramVerniest — APPROVED** [review]
LGTM

File: docs/release-notes/releaseposts/2024042301.md (hunk 1 above)
Comment (BramVerniest) [reply docs/release-notes/releaseposts/2024042301.md]: Fixed it

Comment (seancrowe) [reply docs/release-notes/releaseposts/2024042301.md]: Test

**Pull request merged**